main/main.render_script

```diff
@@ -89,6 +77,19 @@ function update(self)
     render.disable_state(render.STATE_DEPTH_TEST)
     render.disable_state(render.STATE_CULL_FACE)    
     render.draw_debug3d()
+
+    -- SPRITE + LABEL + PARTICLE 3d space
+    --
+    render.set_blend_func(render.BLEND_SRC_ALPHA, render.BLEND_ONE_MINUS_SRC_ALPHA)
+    render.enable_state(render.STATE_DEPTH_TEST)
+    render.enable_state(render.STATE_STENCIL_TEST)
+    render.enable_state(render.STATE_BLEND)
+    render.draw(self.tile_pred)
+    render.draw(self.particle_pred)
+    render.disable_state(render.STATE_BLEND)
+    render.disable_state(render.STATE_STENCIL_TEST)
+    render.disable_state(render.STATE_DEPTH_TEST)
+    render.draw_debug3d()
```

Comment:
I just noticed that we have two render.draw_debug3d(). We should be able to remove the first one right? Perhaps you could include this in the PR?